aten/src/ATen/detail/XPUHooksInterface.h

```diff
@@ -53,9 +56,10 @@ struct TORCH_API XPUHooksInterface {
         XPU_HELP);
   };
 
-  virtual DLDevice getDLPackDeviceFromATenDevice(
+  virtual DLDevice& getDLPackDeviceFromATenDevice(
+      DLDevice& dl_device,
       const Device& aten_device,
-      void* data) const {
+      void* data) const& {
```

Comment:
Why this change is necessary?
```suggestion
      void* data) const {
```